src/qt/test/test_main.cpp

```diff
@@ -54,6 +54,13 @@ int main(int argc, char* argv[])
 
     NodeContext node_context;
     std::unique_ptr<interfaces::Node> node = interfaces::MakeNode(&node_context);
+    gArgs.ForceSetArg("-listen", "0");
```

Comment:
Passing 0 to `port` lets the system assign a free port number. This fixes the issue while allowing for future GUI tests that involve interactions with another node.

```suggestion
    gArgs.ForceSetArg("-port", "0");
```

Reply:
I don't think that is necessary at this time. Additionally, I don't think doing this would be useful. In this hypothetical test, the test framework needs to know what port is being used, so I don't think this would help.

Reply:
true